src-test/org/etools/j1939_84/bus/j1939/packets/DiagnosticTroubleCodeTest.java

```diff
@@ -53,11 +50,11 @@ public void test3() {
     @Test
     public void testEquals() {
         int[] data = new int[] { 0x61, // conversion method
```

Comment:
Is this still used?

Reply:
It includes a test to verify that the "easy encoding" produces a reasonable value. (Assuming "this" refers to the testEquals())

Reply:
no, "this" was referring to the `data` field.